app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cash = {}
+
+    def wrapper(*args, **kwargs) -> int:
```

Comment:
It is not guaranteed that `wrapper` would return `int`, it could return `Any`. Indicate this in the return type annotation.